### PR TITLE
Folding Hasher

### PR DESCRIPTION
**Pull request by ConnorScrums**

### CODE REVIEW

File: folding_method_hasher.py [hunk 1]
```diff
@@ -0,0 +1,22 @@
+from hash import Hash
+from hash_64_bit import Hash64Bit
+from hasher import Hasher
+
+class FoldingMethodHasher(Hasher):
+    def hash(self, bytes: bytes) -> Hash:
+        # Assuming we want to fold the bytes into 8-byte chunks
```
Comment (AnthonyGoncharenko) [inline folding_method_hasher.py:7]: Please run pylint ($git-ls *.py) and fix all warnings/errors.

**Review by AnthonyGoncharenko — COMMENTED** [review]
Work on this so that it fits within the pylint standards

**Comment by ConnorScrums** [comment]
image
got it down to these two.

**Comment by AnthonyGoncharenko** [comment]
> image got it down to these two.

redefining built-in bytes: you can look at what i did (call the parameter raw_bytes)
too few public methods: similarly you can see what i did in mine

**Comment by ConnorScrums** [comment]
Thanks. This should be good to go.

**Review by jnewman530 — APPROVED** [review]
Looks good to me maybe just have Anthony verify the pylint stuff I am not familiar with. I was not able to run the app to test the changes.

**Review by liancent — APPROVED** [review]

**Comment by AnthonyGoncharenko** [comment]
There are conflicts with main, you should fix those before doing the merge of the PR

**Pull request merged**